store/postgres/src/relational_queries.rs

```diff
@@ -1250,17 +1281,19 @@ impl<'a> FilterCollection<'a> {
 /// is `None`, the sort key should be ignored
 #[derive(Debug, Clone)]
 pub struct SortKey {
-    name: Option<SqlName>,
+    column: Option<Column>,
+    value: Option<Value>,
```

Comment:
The code in this file is very careful to do a minimal amount of allocations; I was a little lazy with the `name` (though it was only a string) Now that we are storing more complex data here, we should use references, i.e., this should become
```rust
pub struct SortKey<'a> {
  column: Option<&'a Column>,
  value: Option<&'a Value>
}
```